README.md

```diff
@@ -27,13 +27,15 @@ They are converted to a single RML document.
 Note that the keys in `prefixes`, `sources`, and `mappings` have to be unique across all files.
 `base` can only be set once.
 An example can be found in [`test/multiple-input-files`](test/multiple-input-files).
+You can overwrite external references via the `-e`.
+For example, `-e name=John` will replace all occurences of `$(_name)` with `John`.
+Repeat `-e` for multiple references.
 
 #### yarrrml-generator
 
 If you want to generate YARRRML rules from an RML document, you do the following: `yarrrml-generator -i rules.rml.ttl`.
 The rules will be written to standard output.
 If you want to write them to a file, you can add the `-o` option.
-If you want to use `rr:class` instead of Predicate Object Maps, use the `-c` flag.
```

Comment:
Shouldn't this comment be moved to previous section instead of removed?

Reply:
It's already in the yarrrml-parser section. And here it doesn't work that option, right?

Reply:
ah ok, then it's fine!